hyperbook/book/entwicklung/umgebung.md

```diff
@@ -1,59 +1,18 @@
+index: 10
 ---
 
-# Ablauf des Wettkampfs
-
-Die Software-Challenge wird in drei Phasen ausgespielt.
-
-## Die Meisterschaftsphase
-
-In dieser ersten Phase (Anfang März bis Anfang April)
-wird in Gruppen mit je ca. 20 Teams gespielt.
-
-Die Meisterschaft ist in Spieltagen organisiert.
-An jedem Spieltag hat jedes Team eine Begegnung mit einem anderen Team.
-Bei einer ungeraden Teamanzahl in einer der Gruppen setzt an jedem Spieltag eines der Teams aus.
-
-Vor Beginn des ersten Spieltages soll jedes Team einen funktionsfähigen
-:t[Computerspieler]{#player} über das Online-Abgabesystem eingeschickt haben. Zwischen den
-Spieltagen besteht die Möglichkeit, Clients ebenfalls über dieses System
-zu aktualisieren. Über das Online-Abgabesystem werden auch die
-Ergebnisse, Ranglisten etc. veröffentlicht.
-
-Eine Begegnung gewinnt derjenige Spieler mit den meisten gewonnen
-Spielen in dieser Begegnung. Bei gleich vielen Siegen endet die
-Begegnung unentschieden. Eine gewonnene Begegnung zählt für die
-Rangliste zwei Punkte, ein Unentschieden einen Punkt, eine verlorene
-Begegnung null Punkte.
-
-In der Rangliste wird als nachrangiges Kriterium ausschließlich die
-Anzahl der Siegpunkte aus den bisherigen Spielen verwendet.
-Wenn am Ende einer Meisterschaft zwei Teams denselben Platz belegen,
-entscheidet das Los.
-
-## Die Champions League
-
-In diese zweite Phase (Mitte April bis Mitte Mai)
-kommen die besten acht Teams einer jeden Gruppe aus der ersten Phase.
-Die Champions League wird nach dem gleichen Schema wie eine Meisterschaft ausgespielt.
-
-## Das Final Eight
-
-An der dritten Phase, dem Final Eight, das im Juni in Kiel stattfindet,
-nehmen die besten acht Mannschaften aus der Champions League teil.
-Viertelfinale, Halbfinale, Kleines Finale (= Spiel um den 3. Platz)
-und Finale werden im K.O.-System gespielt.
+# Technische Daten für die Ausführung der Computerspieler
 
-Anders als bei den vorherigen Phasen kann es beim Final Eight keine Begegnung geben,
-die Unentschieden endet.
-Haben beide Teams in einer Begegnung gleich viele Siege errungen,
-wird als nachrangiges Kriterium ausschließlich die Anzahl der am Spielende erreichten Punkte berücksichtigt,
-um den Sieger einer Begegnung zu ermitteln.
+Bei allen im :t[Wettkampfsystem]{#contest} ausgetragenen Spielen
+laufen die :t[Computerspieler]{#player} auf den Servern des Wettkampfsystems.
 
-Im Übrigen gelten beim Final Eight die bekannten Regeln der beiden vorherigen Phasen.
-Es wird dabei auch dieselbe Hardware eingesetzt.
+|                  |                                                   |
+|------------------|---------------------------------------------------|
+| Betriebssystem:  | 64 Bit Linux                                      |
+| Prozessor:       | Ein Kern von einem [Intel Xeon E5-2620 v4, 2,1 GHz](https://ark.intel.com/de/products/92986/Intel-Xeon-Processor-E5-2620-v4-20M-Cache-2_10-GHz) |
+| Arbeitsspeicher: | 1,5 GB                                            |
```

Comment:
We should align / merge this section with https://contest.software-challenge.de/seasons/2024/contestants/2374/clients/new and https://docs.software-challenge.de/entwicklung/abgabe to prevent duplication. I think having it as an own docs page rather than in the contest system is sensible.

![image](https://github.com/software-challenge/docs/assets/13354331/380d35e8-1e18-40de-adb6-834698835615)